script.js

```diff
@@ -1,4 +1,139 @@
+var SCISSORS = "scissors";
+var PAPER = "paper";
+var STONE = "stone";
+var REVERSE_SCISSORS = "reverse scissors";
+var REVERSE_PAPER = "reverse paper";
+var REVERSE_STONE = "reverse stone";
+var numOfUserWins = 0;
+var numOfComputerWins = 0;
+var numOfDraws = 0;
+var numOfRounds = 0;
+var modeAskForUserName = "user types in their name";
+var modeStartPlaying = "sps game starts";
+var gameMode = modeAskForUserName;
+var userName = "";
+
+var programOutput = function () {
```

Comment:
1. Best practice is for us to use a verb when naming these helper functions (e.g. getProgramOutput).
2. Could the function name be better? Currently it seems pretty generic and does the give the reader a sense of what the code does.